soc/nordic/common/uicr/gen_uicr/Kconfig

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: Apache-2.0
+
+source "Kconfig.zephyr"
```

Comment:
nit. non-blocking - might want to put this at bottom so that the application Kconfigs appear at the top